ethereum/blockcreation/src/main/java/org/hyperledger/besu/ethereum/blockcreation/MiningCoordinator.java

```diff
@@ -28,9 +28,25 @@
 
 public interface MiningCoordinator {
 
-  void enable();
+  void start();
 
-  void disable();
+  void stop();
+
+  void awaitStop();
+
+  /**
+   * If mining is enabled, disable it.
+   *
+   * @return True if mining status has transitioned from disabled to enabled
+   */
+  boolean enable();
```

Comment:
Is the javadoc flipped?  @return looks fine, but the text looks like it describes `disable()`